Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -139,13 +138,17 @@ final class DefaultThemeManager: ThemeManager, Notifiable {
 
     // MARK: - Private methods
 
-    private func generateInitialTheme() -> Theme {
+    private func loadInitialThemeType() -> ThemeType {
+        if let nightModeIsOn = userDefaults.object(forKey: ThemeKeys.NightMode.isOn) as? NSNumber,
+           nightModeIsOn.boolValue == true {
+            return .dark
+        }
         var themeType = getSystemThemeType()
-        if let savedThemeDescription = userDefaults.string(forKey: ThemeKeys.themeName),
-           let savedTheme = ThemeType(rawValue: savedThemeDescription) {
+        if let savedThemeDescription = UserDefaults.standard.string(forKey: ThemeKeys.themeName),
```

Comment:
Using user defaults standard suite directly here is just a quick fix to get this in for 106. I was hoping to keep all of the user defaults needed for theming in the one place without creating a migration but it seems the values used by the legacy system come from both the default suite and the custom suite used by prefs.
There are a few options for moving forward here, I would like to have a work session with Laurie when she gets back to figure it out. For now this hacky fix will at least get it working for the release.

Reply:
nit: @OrlaM it would be good to add a todo comment with a ticket but if you already have it as a part of theming work then you can ignore this comment